NewSpring/ApollosProject
# [DEV-3886] Refactor Prayer Menu Tile Cards

## DESCRIPTION

### What does this PR do, or why is it needed?
This PR refactors the prayer menu cards to be shorter and wider than before. It also changes the colors of the selected and deselected states. All this is based off the mocks.

### How do I test this PR?
Make sure you're pointed to `alpha-rock`.
Start up the sim and verify the colors and height/width of the buttons.
Note: You may need to uninstall/reinstall the app on the sim in order to pick up the colors. I noticed sometimes that caching was an issue.

---
> I am affirming this is my _best_ work ([Ecclesiastes 9:10](https://www.bible.com/bible/97/ECC.9.10.MSG))

## TODO


## REVIEW


**Manual QA**


---

> The purpose of PR Review is to _improve the quality of the software._
